common/Environments/Styles/HorizontalCardStyles.xaml

```diff
@@ -61,6 +61,7 @@
         TargetType="Grid">
         <Setter Property="HorizontalAlignment" Value="Stretch" />
         <Setter Property="ColumnSpacing" Value="20" />
+        <Setter Property="Height" Value="36" />

```

Comment:
I assume this causes the Card not to scale with text, which is an accessibility bug. (System settings -> Text Size -> set to 200% and see). Can you use MinHeight here?

Reply:
I used this since I want the height to not change when I show the 'Dot Operations' UI. And it does scale:
![image](https://github.com/microsoft/devhome/assets/16077119/145a3e08-a2d8-4692-b269-e59b556e3a5f)